libs/blocks/marketo/marketo.js

```diff
@@ -18,13 +18,32 @@ import { parseEncodedConfig, loadScript, createTag } from '../../utils/utils.js'
 const FORM_ID = 'form id';
 const BASE_URL = 'marketo host';
 const MUNCHKIN_ID = 'marketo munckin';
+const FORM_MAP = {
+  'destination-url': 'form.success.content',
+  'co-partner-names': 'program.copartnernames',
+  'sfdc-campaign-id': 'program.campaignids.sfdc',
+};
 
 export const formValidate = (form) => {
   const formEl = form.getFormElem().get(0);
   formEl.classList.remove('hide-errors');
   formEl.classList.add('show-warnings');
 };
 
+export const decorateURL = (destination, baseURL = window.location) => {
+  let destinationUrl = new URL(destination, baseURL.origin);
+
+  if (destinationUrl.hostname.includes('.hlx.')) {
+    destinationUrl = new URL(`${destinationUrl.pathname}${destinationUrl.search}${destinationUrl.hash}`, baseURL.origin);
```

Comment:
nit: use destructuring (also shortens line that is > 100)
```suggestion
    const { pathname, search, hash } = destinationUrl;
    destinationUrl = new URL(`${pathname}${search}${hash}`, baseURL.origin);
```

Reply:
Looking at code below you prob want to destructure outside the if

Reply:
Sounds good, thanks!